ServiceTemplate.html

```diff
@@ -26,18 +26,18 @@
     <div class="sidebar-border bwgrad"></div>
   </div>
   <div class=bottom-right-image>
-    <img alt="Home Text" src=../../images/Home.svg>
+    <img alt="Home (doesn't work)" src=../../images/Home.svg>
```

Comment:
You do realize that the sidebar's home text doesn't act as a link, right? Why would you put "(doesn't work)" as the alt text?